WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -845,12 +952,17 @@ - (void)configureCell:(UITableViewCell *)aCell atIndexPath:(NSIndexPath *)indexP
 
 - (CGFloat)tableView:(UITableView *)tableView estimatedHeightForRowAtIndexPath:(NSIndexPath *)indexPath
 {
+    // Note:
+    // Due to a UITableView bug on iOS 8, let's keep the estimated height to the bare minimum.
+    // If the estimated is bigger than needed, UITableView might actually use that size,
+    // and shrink using an undesired animation.
+    
```

Comment:
Should this comment go with the const?  I'm not sure I'd have checked this method and seen the note if I just wanted to update the estimated height and I knew it was in the const.